pre-processing.py

```diff
@@ -0,0 +1,116 @@
+# if kernel crashes, make sure pywin32 and pipywin32 are installed.
+# Followed instructions here: https://github.com/jupyter/notebook/issues/4909
+# import win32api
+
+import pandas as pd
+import numpy as np
+import plotly.graph_objects as go
+import plotly.express as px
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import dash_bootstrap_components as dbc
+import microdf as mdf
+import os
+import us
+
+# Import data from Ipums
+person_raw = pd.read_csv("cps_00041.csv.gz")
```

Comment:
```suggestion
person = pd.read_csv("cps_00041.csv.gz")
```
And remove lines 21-22 (this is really just useful for notebooks), addresses #35